src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -18,6 +19,9 @@
 #include <infiniband/arch.h>
 #include <netinet/in.h>
 #include <string.h>
+#if __ARM_NEON
```

Comment:
it should be inherited; if not need to include a platform-generic header from ucs/arch/XX.h